dags/openshift_nightlies/manifest.yaml

```diff
@@ -25,93 +33,61 @@ dagConfig:
 
 platforms:
   cloud:
-    - version: 4.8
-      releaseStream: 4.8.0-0.nightly
-      providers:
-      - name: aws
-        profiles:
-        - default
-        - ovn
-      - name: azure
-        profiles:
-        - default
-      - name: gcp
-        profiles:
-        - default
-    - version: 4.9
-      releaseStream: 4.9.0-0.nightly
-      providers:
-      - name: aws
-        profiles:
-        - default
-        - ovn
-      - name: azure
-        profiles:
-        - default
-      - name: gcp
-        profiles:
-        - default
-    - version: "4.10"
-      releaseStream: 4.10.0-0.nightly
-      providers:
-      - name: aws
-        profiles:
-        - default
-        - ovn
-        - acs
-      - name: azure
-        profiles:
-        - default
-      - name: gcp
-        profiles:
-        - default    
-  baremetal:
-    - version: 4.7
-      releaseStream: latest-4.7
-      build: dev
-      profiles:
-      - jetski  
-    - version: 4.8
-      releaseStream: latest-4.8
-      build: dev
-      profiles:
-      - jetski
-    - version: 4.9
-      releaseStream: latest-4.9
-      build: dev
-      profiles:
-      - jetski      
+    versions: [4.9, "4.10"]
```

Comment:
can we separate out the versions , since we need different variants for different versions. eg: we don't need medium and large for 4.10. this is only for aws.